src/posix/client.cpp

```diff
@@ -164,7 +164,7 @@ static void SendBlockingCommand(int aArgc, char *aArgv[])
         ssize_t rval = read(sSessionFd, buffer, sizeof(buffer));
 
         VerifyOrExit(rval >= 0);
-        write(STDOUT_FILENO, buffer, static_cast<size_t>(rval));
+        IgnoreReturnValue(write(STDOUT_FILENO, buffer, static_cast<size_t>(rval)));
```

Comment:
@dismirlian, thanks for addressing this (again). This seems to be a follow-on to #4480.

Note that the need to properly handle `write()` return values was previously noted in https://github.com/openthread/openthread/pull/4480#discussion_r367690447

And as noted, we should have a separate PR for addressing  those issues.

Reply:
Hi @jwhui, yes in fact I wanted to re-surface this issue. If you want, I could prepare a PR addressing this.

Reply:
@dismirlian, that would be great!

Reply:
@jwhui please see #4580